test/e2e/tests/hardware-wallets/trezor-account.spec.ts

```diff
@@ -0,0 +1,117 @@
+import FixtureBuilder from '../../fixture-builder';
+import { withFixtures } from '../../helpers';
+import { shortenAddress } from '../../../../ui/helpers/utils/util';
+import { KNOWN_PUBLIC_KEY_ADDRESSES } from '../../../stub/keyring-bridge';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import ConnectHardwareWalletPage from '../../page-objects/pages/hardware-wallet/connect-hardware-wallet-page';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import HomePage from '../../page-objects/pages/home/homepage';
+import SelectTrezorAccountPage from '../../page-objects/pages/hardware-wallet/select-trezor-account-page';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+
+describe('Trezor Hardware', function () {
+  it('derives the correct accounts and unlocks the first account', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }) => {
+        await loginWithBalanceValidation(driver);
+
+        const headerNavbar = new HeaderNavbar(driver);
+        await headerNavbar.openAccountMenu();
+
+        // Choose connect hardware wallet from the account menu
+        const accountListPage = new AccountListPage(driver);
+        await accountListPage.check_pageIsLoaded();
+        await accountListPage.openConnectHardwareWalletModal();
+
+        const connectHardwareWalletPage = new ConnectHardwareWalletPage(driver);
+        await connectHardwareWalletPage.check_pageIsLoaded();
+        await connectHardwareWalletPage.openConnectTrezorPage();
+
+        const selectTrezorAccountPage = new SelectTrezorAccountPage(driver);
+        await selectTrezorAccountPage.check_pageIsLoaded();
```

Comment:
Noticed this setup code is repeated and isn’t specific to this test. What do you think about extracting it into a setup method, maybe something like `openSelectTrezorAccountPage`, or even a reusable flow?

Reply:
Thanks for the suggestion Derek. I am currently focusing on creating base pages and methods for hardware wallet pages. For the next steps to better organize the tests, I will leave it to hardware wallet team to decide how to better extract based on their needs.